indico/modules/events/surveys/controllers/display.py

```diff
@@ -18,20 +18,24 @@
 
 from flask import redirect, flash, session, request
 from sqlalchemy.orm import joinedload
+from werkzeug.datastructures import MultiDict
 from werkzeug.exceptions import Forbidden
 
 from indico.core.db import db
+from indico.core.db.sqlalchemy.util.session import no_autoflush
 from indico.modules.auth.util import redirect_to_login
 from indico.modules.events.models.events import EventType
 from indico.modules.events.surveys.models.submissions import SurveyAnswer, SurveySubmission
 from indico.modules.events.surveys.models.surveys import Survey, SurveyState
-from indico.modules.events.surveys.util import make_survey_form, was_survey_submitted, save_submitted_survey_to_session
+from indico.modules.events.surveys.util import (make_survey_form, is_submission_in_progress,
+                                                save_submitted_survey_to_session, was_survey_submitted)
 from indico.modules.events.surveys.views import (WPDisplaySurveyConference, WPDisplaySurveyMeeting,
                                                  WPDisplaySurveyLecture)
+from indico.legacy.webinterface.rh.conferenceDisplay import RHConferenceBaseDisplay
```

Comment:
keeping indico.legacy separate is actually not too bad (like we did with makac).

however, considering that (AFAIK) none of the tools that sort imports let you enforce that, i guess going for alphabetical order whenever we add a legacy import is easier